apps/patient/src/components/PharmacyCard.tsx

```diff
@@ -142,27 +138,15 @@ export const PharmacyCard = memo(function PharmacyCard({
                 <TagLabel> {t.preferred}</TagLabel>
               </Tag>
             ) : null}
-            {previous && !preferred ? (
-              <Tag size="sm" colorScheme="green">
-                <TagLeftIcon boxSize="12px" as={FiRotateCcw} />
-                <TagLabel> {t.previous}</TagLabel>
-              </Tag>
-            ) : null}
-            {goodService ? (
-              <Tag size="sm" colorScheme="purple">
-                <TagLeftIcon boxSize="12px" as={FiThumbsUp} />
-                <TagLabel> {t.goodService}</TagLabel>
-              </Tag>
-            ) : null}
-            {pharmacy?.is24Hr ? (
-              <Tag size="sm" colorScheme="green">
-                <TagLabel>{t.open24hrs}</TagLabel>
+            {pharmacy?.isUrgent ? (
+              <Tag size="sm" bgColor="yellow.200">
+                <TagLabel>Ready in 30 minutes</TagLabel>
               </Tag>
             ) : null}
           </HStack>
           <VStack align="start" w="full" spacing={0}>
             <HStack spacing={2}>
-              {pharmacy?.logo ? <Image src={pharmacy.logo} width="auto" height="24px" /> : null}
+              {pharmacy?.logo ? <Image src={pharmacy.logo} width="auto" height="20px" /> : null}
```

Comment:
Costco logo has white border, wags doesn't so it looked quite big. Double-checked, costco still looks good.